blah.md

```diff
@@ -0,0 +1,34 @@
+Note on the database layout of `metrics` and `metrics_rollups`
```

Comment:
Uh... is this supposed to be here?  Seems like this was a temp file for the comment above, correct?

Reply:
Seems like you have a couple of files that are like this.